testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -260,13 +260,16 @@ def test_check_coordinate_limits_max_noconversion(self):
 
 
 class TestGROWriterLarge(TestCase, tempdir.TempDir):
-    def setUp(self):
-        self.tmpdir = tempdir.TempDir()
-        self.large_universe = mda.Universe(GRO_large)
 
-    def tearDown(self):
-        del self.tmpdir
-        del self.large_universe
+    @classmethod
```

Comment:
Note: I don't think that it is save to use class-level setup for Universes in general. If you have parallel tests that iterate over a trajectory you will get cross interactions because you start sharing state.

In *this particular case* where only a single frame is read it *might* be fine but please add a comment saying that you carefully considered making this a class level set up.